### PR TITLE
Feature/34 firebase socials auth

### PR DESCRIPTION
**Pull request by Limbo2332**

### CODE REVIEW

**Comment by Limbo2332** [comment]
https://github.com/BinaryStudioAcademy/bsa-2023-leetwars/assets/93576084/95f16cd0-3c5a-4555-be6d-c8c9a881a744


https://github.com/BinaryStudioAcademy/bsa-2023-leetwars/assets/93576084/5346c3ba-383a-40c4-bb13-129446e13832

**Comment by Limbo2332** [comment]
[Firebase socials auth](https://trello.com/c/rnEWW2L5/34-firebase-socials-auth)

**Review by apofis96 — APPROVED** [review]

File: backend/LeetWars.Core/LeetWars.Core.Common/DTO/User/UserLoginDto.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿namespace LeetWars.Core.Common.DTO.User
+{
+    public class UserLoginDto
+    {
```
Comment (tatianahutii) [inline backend/LeetWars.Core/LeetWars.Core.Common/DTO/User/UserLoginDto.cs:4]: where do we use this class?

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: backend/LeetWars.Core/LeetWars.Core.Common/DTO/User/UserLoginDto.cs (hunk 1 above)
Comment (Limbo2332) [reply backend/LeetWars.Core/LeetWars.Core.Common/DTO/User/UserLoginDto.cs:4]: I used only in frontend part for now, so I thought to add it in backend

Comment (tatianahutii) [reply backend/LeetWars.Core/LeetWars.Core.Common/DTO/User/UserLoginDto.cs:4]: I believe we don't need it, as we do not send password to our backend

**Review by tatianahutii — APPROVED** [review]

**Pull request merged**